File: Conditonal&SwitchCase/C/CurrencyNotes.c
#include <stdio.h>

int main() {
    int amount;

    printf("Enter the amount: ");
    scanf("%d", &amount);

    printf("Number of notes required for Rs. %d:\n", amount);
    printf("500 Rs notes: %d\n", amount / 500); amount = amount % 500;
    printf("200 Rs notes: %d\n", amount / 200);  amount = amount % 200;
    printf("100 Rs notes: %d\n", amount / 100);  amount = amount % 100;
    printf("50 Rs notes: %d\n", amount / 50);   amount = amount % 50;
    printf("20 Rs notes: %d\n", amount / 20);   amount = amount % 20;
    printf("10 Rs notes: %d\n", amount / 10);   amount = amount % 10;
    printf("5 Rs notes: %d\n", amount / 5);  amount = amount % 5;
    printf("2 Rs notes: %d\n", amount / 2);   amount = amount % 2;
    printf("1 Rs notes: %d\n", amount / 1);
    
    return 0;
}

File: Conditonal&SwitchCase/java/CurrencyNotes.java

import java.util.Scanner;

public class CurrencyNotes {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter Your amount");
        int amount = sc.nextInt();

        System.out.println("500 Notes : "+amount/500); amount%=500;
        System.out.println("200 Notes : "+amount/200); amount%=200;
        System.out.println("100 Notes : "+amount/100); amount%=100;
        System.out.println("50 Notes : "+amount/50); amount%=50;
        System.out.println("20 Notes : "+amount/20); amount%=20;
        System.out.println("10 Notes : "+amount/10); amount%=10;
        System.out.println("5 Notes : "+amount/5); amount%=5;
        System.out.println("2 Notes : "+amount/2); amount%=2;
        System.out.println("1 Notes : "+amount/1); amount%=1;
    }
}
